### PR TITLE
Create LDAP CI test

### PR DESCRIPTION
**Pull request by RonnyFrayRegato**
Create a continuous integration pipeline for CWS using LDAP security and associated tests.

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -82,16 +107,53 @@ jobs:
...
             }
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+
+  publish-cws-image:
+
+    if: contains(github.event.head_commit.message, 'version')
+    env:
+      CWS_VERSION: "2.3.0-pre.1"  # update this each CWS release
```
Comment (jamesfwood) [inline .github/workflows/maven.yml:129]: Why is this needed?  Shouldn't it pull the version from the tag?

Comment (RonnyFrayRegato) [reply .github/workflows/maven.yml:129]: Yes, I'll make the relevant changes

Comment (RonnyFrayRegato) [reply .github/workflows/maven.yml]: The `github.ref_name` context variable contains the branch or tag name that triggered the workflow run, which will be used in the Docker image tag when the CD portion is triggered.
